uiowa/uiowa
# Image gallery

Resolves #4827 

# How to test

1. blt ds --site=sandbox.uiowa.edu
2. https://sandbox.prod.drupal.uiowa.edu/single-image-gallery
3. Check to make sure that the node image styles is "medium - no crop" and the Photoswipe image is the original image.
